tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "esModuleInterop": true,
```

Comment:
I moved this setting from `packages/github/tsconfig.json` to here.

It's needed for the GitHub unit tests to work. Otherwise the base class properties don't work.

For example, without this setting the unit test fails attempting to access `repos` property fails:

```js
    const octokit = new GitHub(token)
    const branch = await octokit.repos.getBranch({
      owner: 'actions',
      repo: 'toolkit',
      branch: 'master'
    })
```

Typescript [\"highly recommend applying it\"](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html)

Reply:
@hross thoughts regarding consumer impact. I'm assuming shouldnt be any

Reply:
I think it's fine.